tests/unit/meta/test_application.py

```diff
@@ -64,33 +64,6 @@ def test_app_no_change(self):
         self.expectThat("command-foo.wrapper", Not(FileExists()))
         self.expectThat("stop-command-foo.wrapper", Not(FileExists()))
 
-    def test_mangling(self):
```

Comment:
Not saying it is wrong to remove, but what is the rationale for removing them?

Reply:
Oh it is wrong to remove ;) Sorry, I meant to go back and reconsider these tests, but forgot!  I originally thought they were testing things that were already covered elsewhere and didn't make sense to me by name.   Looking at it again, I see they actually should remain, but rename mangling -> massaging.  It's really testing that $SNAP is stripped for this case for core18, but not anything outside of massaged bases (e.g. core20).

In actu